CHANGELOG.md

```diff
@@ -4,6 +4,8 @@
 
 * C#
   * Add support for Labeled Strings and Labeled Booleans.
+* Python
+  * Breaking change: `data_dir` must always be passed to `Glean.initialize`. Prior to this, a missing value would store Glean data in a temporary directory.
```

Comment:
Previously we used `**Breaking change**` just to make it stand out a bit more when rendered.

```suggestion
  * **Breaking change**: `data_dir` must always be passed to `Glean.initialize`. Prior to this, a missing value would store Glean data in a temporary directory.
```